.evergreen/config.yml

```diff
@@ -1385,31 +1385,39 @@ tasks:
     commands:
       - func: run-make
         vars:
-          targets: "build"
+          targets: "build build-tests"
           BUILD_ENV: "PATH=/opt/golang/go1.10/bin:$PATH GOROOT=/opt/golang/go1.10"
 
+  # Build with whatever the latest Go version is that we're using for tests
```

Comment:
We're introducing tests that are build-flagged based on Go versions because they use Go features that were added in 1.13, so we need this new task because those tests aren't covered by other build tasks.